ui/app/templates/components/auth-jwt.hbs

```diff
@@ -46,22 +46,25 @@
   <div data-test-yield-content>
     {{yield}}
   </div>
-  <button
-    data-test-auth-submit={{true}}
-    type="submit"
-    disabled={{@disabled}}
-    class="button is-primary {{if @disabled 'is-loading'}}"
-    id="auth-submit"
-  >
-    {{#if this.isOIDC}}
-      {{#if this.role.providerButtonComponent}}
-        {{component this.role.providerButtonComponent}}
-      {{else}}
-        Sign in with
-        {{or this.role.providerName "OIDC Provider"}}
-      {{/if}}
-    {{else}}
-      Sign In
-    {{/if}}
-  </button>
+
+  {{#if this.isOIDC}}
+    <Hds::Button
+      @text={{concat "Sign in with " (or this.role.providerName "OIDC Provider")}}
+      @loading={{@disabled}}
```

Comment:
```suggestion
```

Reply:
I don't see this in the docs as an arg? Is it supposed to be a loading icon if a save is running?